scenarioo-server/src/main/java/org/scenarioo/business/diffViewer/comparator/ScreenshotComparator.java

```diff
@@ -39,9 +37,8 @@
 	private static final int SCREENSHOT_DEFAULT_CHANGE_RATE = 0;
 	private DiffReader diffReader;
 
-	public ScreenshotComparator(final String baseBranchName, final String baseBuildName,
-								final ComparisonConfiguration comparisonConfiguration) {
-		super(baseBranchName, baseBuildName, comparisonConfiguration);
+	public ScreenshotComparator(ComparatorParameter parameter) {
```

Comment:
`parameter` should be renamed to `parameters`everywhere since it is a collection of several parameters.